roles/openshift_health_checker/library/aos_version.py

```diff
@@ -78,7 +82,7 @@ def bail(error):  # pylint: disable=missing-docstring
         for name in not_found:
             msg += '  %s\n' % name
         bail(msg + 'Please check your subscriptions and enabled repositories.')
-    if multi_found:
+    if multi_found and rpm_prefix == 'atomic-openshift':
```

Comment:
The check for multiple minor versions present is really only relevant for enterprise. I assume this merits a comment... I only explained in the commit msg